packages/navie/package.json

```diff
@@ -45,6 +45,7 @@
     "@langchain/openai": "^0.2.7",
     "fast-xml-parser": "^4.4.0",
     "js-yaml": "^4.1.0",
+    "jsdom": "^16.6.0",
     "langchain": "^0.2.16",
     "mermaid": "^9",
```

Comment:
As long as you're here, can you take a look at upgrading to mermaid 10? I believe that would allow vscode-appland to link to a local appmap-js repo.

Reply:
I actually did try, but mermaid 10 is ESM-only.